Garcia-Julz/mine-your-business
# Jg views 1

# Description
The views are now added and there is functionality to where you made be able to see lists of miners, rigs, and tickets. Also you are able to login and log out with certain links

Closes #[Enter issue #]
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-views-1`
4. Once you are on the brank go to the room directory and play around with teh orls suck as http://localhost:8000/tickets/ to views all lists
# Checklist:
